spec/support/with_project_cleanup.cr

```diff
@@ -0,0 +1,18 @@
+module WithProjectCleanup
+  private def with_project_cleanup(project_directory = "test-project", skip_db_drop = false)
+    yield
+
+    FileUtils.cd(project_directory) {
+      output = IO::Memory.new
+      Process.run(
+        "lucky db.drop",
+        output: output,
+        shell: true
+      )
+
+      output.gets_to_end.should contain("Done dropping")
```

Comment:
This spec fails for me locally. Says `output` is `""`. No errors happen. The result code is 0. I even added a `gets` to pause the spec and run it manually, and it runs fine manually. My guess is maybe something with `Process` and waiting on crystal to compile? 🤔

Reply:
Alright, so the same spec fails here too. I'm just not sure why or how to fix it.... I'll keep working at that.